db/migrate/20170812153137_add_foreign_key_applicant_id_to_notifications.rb

```diff
@@ -0,0 +1,20 @@
+class AddForeignKeyApplicantIdToNotifications < ActiveRecord::Migration
+  def change
+    reversible do |dir|
+      dir.up do
+        execute <<-SQL
+          ALTER TABLE notifications
+            ADD CONSTRAINT fk_applicant_id
```

Comment:
Do these constraint names not need to be unique? I typically see them named `fk_<table>_<column>`